src/tests/v3-api.test.ts

```diff
@@ -1,163 +0,0 @@
-import test from 'tape';
-
-// import penumbra from '../API';
-import {
-  PenumbraAPI,
-  PenumbraReady,
-  ProgressEmit,
-  RemoteResource,
-} from '../types';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-import penumbra from '../API';
-import { hash, timeout } from './helpers';
-import { TimeoutManager } from './helpers/timeout';
-
-const view = self;
-
-const onReady = async (event?: PenumbraReady) => {
```

Comment:
whats up with these?

Reply:
This is an old testing file that we no longer need (for previous version of our API)